client/src/theme/overrides/Button.js

```diff
@@ -105,6 +105,28 @@ export default function Button(theme) {
             opacity: 0.5,
           },
         },
+        gray: {
+          padding: "6px 12px",
+          color: '#313233',
+          backgroundColor: theme.palette.common.white,
+          borderStyle: "solid",
+          borderWidth: "1px",
+          borderColor: '#999999',
+          fontSize: ".8125rem",
+          "@media (min-width:600px)": {
+            fontSize: "0.8125rem",
+          },
+          "&:hover": {
+            backgroundColor: '#E1E1E1',
+          },
+          "&:active": {
```

Comment:
The active state is not listed in Figma. @JohnHaoHuang, can you clarify the active state?